### PR TITLE
Add folder endpoints

### LINKED ISSUE #7: Folder endpoints
- [ ] Get Folder
- [ ] Create folder
- [ ] Delete folder

### PR DESCRIPTION
**Pull request by vutoan1245**
Resolve #7 

- Get folder
- Create folder
- Update folder
- Delete folder

### CODE REVIEW

File: backend/model/Folder.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Folder.init(
+    {
+        name: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
+        userId: {
+            type: DataTypes.STRING,
+            allowNull: false,
+        },
```
Comment (ljoks) [inline backend/model/Folder.ts:15]: add courseId Field

File: backend/serverless.yml [hunk 2]
```diff
@@ -32,6 +32,7 @@ plugins:
   - serverless-dotenv-plugin
   - serverless-webpack
   - serverless-offline
+  - serverless-dotenv-plugin
```
Comment (ljoks) [inline backend/serverless.yml:35]: remove copy of `serverless-dotenv-plugin`

File: backend/src/folder.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
+import { APIGatewayEvent, Context, ProxyResult } from 'aws-lambda'
+const Folder = require('../model/Folder')
+
+const mockUserid = 1
+const mockCourseId = 1
+
+const getFolder = async (): Promise<ProxyResult> => {
+    return Folder.findAll({
+        where: {
+            userId: mockUserid,
```
Comment (ljoks) [inline backend/src/folder.ts:10]: get folder for courseId, not userId. professor could have multiple courses

**Review by ljoks — CHANGES_REQUESTED** [review]
I added a few comments. please let me know if you want to talk about it any of it, otherwise we can merge as soon as you fix

File: backend/src/folder.ts (hunk 3 above)
Comment (vutoan1245) [reply backend/src/folder.ts:11]: Both courseId and userId for access permission.

**Review by ljoks — APPROVED** [review]

**Pull request merged**